pkg/types/cose/v0.0.1/entry.go

```diff
@@ -316,19 +316,25 @@ func (v V001Entry) CreateFromArtifactProperties(_ context.Context, props types.A
 		}
 	}
 	publicKeyBytes := props.PublicKeyBytes
-	if publicKeyBytes == nil {
-		if props.PublicKeyPath == nil {
+	if len(publicKeyBytes) == 0 {
+		if len(props.PublicKeyPath) == 0 {
 			return nil, errors.New("public key must be provided to verify signature")
 		}
-		publicKeyBytes, err = ioutil.ReadFile(filepath.Clean(props.PublicKeyPath.Path))
-		if err != nil {
-			return nil, fmt.Errorf("error reading public key file: %w", err)
+		if len(props.PublicKeyPath) > 1 {
```

Comment:
again, could consolidate the `>1` and `==1` check

Reply:
done